src/main/java/racingcar/model/Distance.java

```diff
@@ -1,6 +1,6 @@
 package racingcar.model;
 
-public final class Distance {
+public final class Distance implements Comparable {
```

Comment:
Raw Type으로 인터페이스를 활용하시기보다는 Comparable 제네릭 그대로 활용하는게 좋습니다.
Raw Type이나 Object를 그대로 활용하시게 되면 런타임에 비즈니스가 예외가 발생하는 경우가 생길 수 있습니다.
제네릭을 사용하시면 컴파일 타임에 타입에 대한 검증이 끝나고 비즈니스 구현이 단조로워지실 겁니다.
```suggestion
public final class Distance implements Comparable<Distance> {
```
위 처럼 수정하시고 compareTo도 재정의를 다시 한 번 해보시는걸 추천드립니다. 😄

Reply:
아아 ! Comparable 구현시 제네릭을 사용하니 `distance.compareTo(다른 클래스)` 와 같이 잘못된 사용이 컴파일타임에 검증되고 compareTo메서드에 Object에서 Distance로 값을 가져오는 과정도 생략되네요..!!
안그래도 오버라이딩 메서드 인자가 Object인 것이 이상했는데 ㅠㅠ compareTo메서드에 다른 값을 넣어보는 테스트를 했다면 눈치챘을 텐데 너무 당연하다고 생각하고 넘어간 것 같습니다😢 조심하겠습니다!! 감사합니다!!